src/main/resources/roleconfig/role_iac.json

```diff
@@ -901,5 +901,467 @@
         }
       }
     ]
+  },
+  {
+    "name": "specific-access-judiciary",
+    "label": "Specific Access (judiciary)",
+    "description": "Specific access for judicial users.",
+    "category": "JUDICIAL",
+    "substantive": false,
+    "patterns": [
+      {
+        "roleType": {
+          "mandatory": true,
+          "values": ["CASE"]
+        },
+        "grantType": {
+          "mandatory": true,
+          "values": ["BASIC"]
+        },
+        "classification": {
+          "mandatory": true,
+          "values": ["PRIVATE"]
+        },
```

Comment:
This will be retained as SPECIFIC and RESTRICTED as given in Jon's JIRA ticket..that change was only for -granted and -denied.